ruler-gradle-plugin/src/main/kotlin/com/spotify/ruler/plugin/report/JsonReporter.kt

```diff
@@ -39,14 +39,16 @@ class JsonReporter {
      * @param components Map of app component names to their respective files
      * @param ownershipInfo Optional info about the owners of components.
      * @param targetDir Directory where the generated report will be located
+     * @param omitFileBreakdown If true, the list of files for each component and dynamic feature will be omitted
      * @return Generated JSON report file
      */
     fun generateReport(
         appInfo: AppInfo,
         components: Map<DependencyComponent, List<AppFile>>,
         features: Map<String, List<AppFile>>,
         ownershipInfo: OwnershipInfo?,
-        targetDir: File
+        targetDir: File,
+        omitFileBreakdown: Boolean,
```

Comment:
With this new parameter, `generateReport` doesn't meet `LongParameterList` Detekt rule. I was thinking about wrapping `targetDir` and `omitFileBreakdown` params into a new `ReportOutputConfig` class or something like that. WDYT